src/main/java/Database.java

```diff
@@ -0,0 +1,318 @@
+import java.sql.*;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Database {
+
+    public static void main(String[] args) {
+
+        try {
+            Database db = new Database();
+            db.connect();
+            db.clear_all();  // reset everything
+            db.disconnectDB();
+        } catch (SQLException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    private Connection conn = null;
+
+    void clear_all() {
+        clear_schedules();
+        clear_users();
+    }
+
+    boolean connect() {
+        try {
+            // create variable that allows us to pass username and password to DB
```

Comment:
Are these to be unimplemented, or just unfinished

Reply:
I will probably erase those. Those  might be useful if we did an actual SQL database, but we don't need it for SQlite.